### ISSUE TITLE
ssn:ext proposal comment: Note on 4.2.4

### ISSUE THREAD

**Issue by smrgeoinfo** [w3c/sdw#1153]
Guarded constraint text :

> This constraint says that at least one ultimate feature-of-interest is expected for each individual sample. More than one ultimate feature-of-interest may apply when a sample is used in multiple investigations to proxy for different features.

I'm guessing copied from the previous section... should read (?):

> This constraint says that at least one sampled feature is expected for each individual sample. More than one sampled feature may apply when a sample is assembled [aggregated?] from multiple source samples.

**Comment by dr-shorthair** [comment]
Good catch Steve - the name of this property was changed relatively late to align better with the ISO 19156 revision underway. 

However, this constraint is intended to support the case where a single sample is representative of more than one ultimate FOI, e.g. a thin section represents a hand-specimen and also a Geological unit. etc. 

Without getting too wordy, how does this read - 

      This constraint says that at least one sampled feature is expected for each individual sample. 
     More than one sampled feature may apply when a single sample proxies for different ultimate features when used in multiple investigations.

**Comment by smrgeoinfo** [comment]
Isn't sosa:hasUltimateFeatureOfInterest the link to different ultimate FOI? The definition currently in sosa:hasSampledFeature seems like a copy of an earlier version of the definition of sosa:hasUltimateFeatureOfInterest,  perhaps it needs to be updated as well.  Isn't sosa:hasSampledFeature a property to link samples in a sample processing chain?  I was thinking of the situation where, for example a mineralized site is characterized by aggregating (e.g. average) assays from multiple samples at the site, so the analyzed sample proximate sampled feature is a set of samples. The ultimate FOI might be the site (if you're prospecting), or a known mineral deposit (if you're more advanced in an exploration program).

**Comment by dr-shorthair** [comment]
See updated text at https://raw.githack.com/w3c/sdw/simon-ssn-ext/proposals/ssn-extensions/index.html#has-sampled-feature

Note that `sosa:hasUltimateFeatureOfInterest` links from an **activity** (Observation, Sampling, Actuation) - the purple boxes in [Figure 3](https://w3c.github.io/sdw/proposals/ssn-extensions/#fig-sample-chain-geology) - to a FOI. 
And `sosa:hasSampledFeature` links from a Sample (**entity** in Prov-speak) to its original predecessor, typically via a chain of `sosa:isSampleOf` edges. 
`sosa:hasOriginalSample`, links from a Sample (**entity** in Prov-speak) to a key predecessor, alsovia a chain of `sosa:isSampleOf` edges. but not necessarily all the way to the end if you don't want it to - see the diagram [Figure 3](https://w3c.github.io/sdw/proposals/ssn-extensions/#fig-sample-chain-geology).

**Comment by smrgeoinfo** [comment]
Since there are no domain constraints, doesn't the model allow a sample to have a hasUltimateFeatureOfInterest property, or an activity to have a hasSampledFeature property.

**Comment by dr-shorthair** [comment]
Good catch @smrgeoinfo - fixed now. 
See https://raw.githack.com/w3c/sdw/simon-ssn-ext/proposals/ssn-extensions/index.html#properties and https://github.com/w3c/sdw/blob/gh-pages/proposals/ssn-extensions/rdf/ssn-ext.ttl#L154 etc

**Comment by smrgeoinfo** [comment]
I see that the domain and range constraints are implemented in the turtle encoding using Schema.org property schema:domainIncludes and schema:rangeIncludes.  The schema.org docs say the domain of these is schema:Property and the range is schema:Class, but if I understand schema.org these not required.  It seems that using domainIncludes and rangeIncludes will also avoid inferencing the type of the subject or object of these properties by owl parsers as well. 
Is the comment in section 4.2.1 starting "The global domain and range constraints mean that any..." still valid?  Perhaps a note could be added on the properties with domain and range constraints noting that they are implemented using schema.org properties and thus have no implications for owl inferencing. 

something like 

> Domain and range constraints are implemented using schema.org domainIncludes and rangeIncludes properties which will have no entailments under OWL inferencing.

**Comment by dr-shorthair** [comment]
They are in the turtle in both schema.org (following the precedent of SOSA) and now also in rdfs (following the precedent of SSN). They are in the branch so far which I have not merged with the master yet. Following the links above.
